requirements.txt

```diff
@@ -1,6 +1,6 @@
 arrow>=0.8.0,<1.0.0
 importlib-metadata>=2.1.1,<3.0.0; python_version <= '3.5'
-importlib-metadata>=3.3.0,<4.0.0; python_version > '3.5' and python_version < '3.8'
+importlib-metadata>=3.3.0; python_version > '3.5'
```

Comment:
I believe we should keep `and python_version < '3.8'` as from Python 3.8 `importlib.metadata` is part of the standard library so we won't use `importlib-metadata` anyway:

```python
try:
    from importlib.metadata import version
except ImportError:  # ModuleNotFoundError is not available in Python 3.5
    from importlib_metadata import version
```

As you see if `importlib.metadata` exists (3.8+) we never import `importlib_metadata` so no need to install it.

Reply:
Makes sense, I'll update it.